tests/test_collection.py

```diff
@@ -614,6 +614,28 @@ def test_read(self):
                 pass
 
 
+class CollectionTestToFromJson(unittest.TestCase):
+
+    def setUp(self):
+        self._tmp_dir = TemporaryDirectory(prefix='signac_collection_')
+        self._fn_collection = os.path.join(self._tmp_dir.name, 'test.txt')
+        self._fn_json = os.path.join(self._tmp_dir.name, 'test.json')
+        self.addCleanup(self._tmp_dir.cleanup)
+        with open(self._fn_collection, 'w') as file:
+            for i in range(3):
+                file.write('{"a": 0}\n')
```

Comment:
I'd prefer to see the collection built using the normal Collection API for doing this, instead of manually writing the lines. We can rely on that behavior to work, and then this test is specifically focused on the JSON reading/writing.

Reply:
In there an existing test I can consult for making this change? I was basing this on the FileCollectionBadJson test as this style is different from what I'm accustomed to.

Reply:
This might work (untested):

```python
    def setUp(self):
        self.c = Collection.open(filename=':memory:')
        docs = [dict(_id=str(i)) for i in range(10)]
        self.c.update(docs)
        self.c.flush()

   def test_write_and_read(self):
        self.c.to_json(self._fn_json)
        self.assertGreater(os.path.getsize(self._fn_json), 0)
        c = Collection.read_json(self._fn_json)
        self.assertEqual(len(list(c)), 10)
        self.assertEqual(len(c.find()), 10)
        c.close()
```